statsmodels/tsa/tests/test_arima.py

```diff
@@ -41,40 +41,43 @@
 
 
 def test_compare_arma():
-    #this is a preliminary test to compare arma_kf, arma_cond_ls and arma_cond_mle
-    #the results returned by the fit methods are incomplete
-    #for now without random.seed
+    # this is a preliminary test to compare arma_kf, arma_cond_ls
+    # and arma_cond_mle
+    # the results returned by the fit methods are incomplete
+    # for now without random.seed
 
     np.random.seed(9876565)
     x = fa.ArmaFft([1, -0.5], [1., 0.4], 40).generate_sample(nsample=200,
-            burnin=1000)
+                                                             burnin=1000)
 
     # this used kalman filter through descriptive
-    #d = ARMA(x)
-    #d.fit((1,1), trend='nc')
-    #dres = d.res
+    # d = ARMA(x)
+    # d.fit((1,1), trend='nc')
+    # dres = d.res
 
-    modkf = ARMA(x, (1,1))
-    ##rkf = mkf.fit((1,1))
-    ##rkf.params
+    modkf = ARMA(x, (1, 1))
+    # rkf = mkf.fit((1,1))
+    # rkf.params
     reskf = modkf.fit(trend='nc', disp=-1)
     dres = reskf
 
     modc = Arma(x)
```

Comment:
getting this into a separate module would be nice, since this Arma class needs to DIAF.

Reply:
Have worked it over more than once, I think the plan forward should be to thinly wrap `SARIMAX` to produce `ARMA` and `ARIMAX` that simplify the inputs (remove seasonal, and integration for ARMA). These will need to live in statespace, and then a FutureWarning can be added to `ARMA` and `ARIMA`.  Less sure about AR, since a simple OLS AR is very useful for performance.